Examples/Detectors/Geant4Detector/src/MockupSectorBuilder.cpp

```diff
@@ -146,44 +147,22 @@ ActsExamples::MockupSectorBuilder::BuildChamber(
 }
 
 std::shared_ptr<Acts::Experimental::DetectorVolume>
-ActsExamples::MockupSectorBuilder::BuildSector(
+ActsExamples::MockupSectorBuilder::buildSector(
     std::vector<std::shared_ptr<Acts::Experimental::DetectorVolume>>
         det_volumes,
```

Comment:
A lot of the naming conventions are not followed. 
`det_volumes` should be `detVolumes`, as so on for others...

Reply:
Should I also change the names of m_cfg to mCfg , gml_path etc?

Reply:
for member variable we have the convention to start with `m_`. other that that it should be camelCase I think. but you will see snake_case from time to time which is also fine I think but camelCase should be preferred for new code